dashboard/v1.1/src/pages/Config/components/MaterialTable.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { forwardRef, MutableRefObject, useState } from 'react';
+import MaterialTable from 'material-table';
+
+import AddBox from '@material-ui/icons/AddBox';
+import ArrowDownward from '@material-ui/icons/ArrowDownward';
+import Check from '@material-ui/icons/Check';
+import ChevronLeft from '@material-ui/icons/ChevronLeft';
+import ChevronRight from '@material-ui/icons/ChevronRight';
+import Clear from '@material-ui/icons/Clear';
+import DeleteOutline from '@material-ui/icons/DeleteOutline';
+import Edit from '@material-ui/icons/Edit';
+import FilterList from '@material-ui/icons/FilterList';
+import FirstPage from '@material-ui/icons/FirstPage';
+import LastPage from '@material-ui/icons/LastPage';
+import Remove from '@material-ui/icons/Remove';
+import SaveAlt from '@material-ui/icons/SaveAlt';
+import Search from '@material-ui/icons/Search';
+import ViewColumn from '@material-ui/icons/ViewColumn';
+
+const SearchTable = (props: any) => {
+  const [selectedRow, setSelectedRow] = useState(null);
+
+  const tableIcons = {
+    Add: forwardRef(
```

Comment:
too much content in the component. Pls try declaring this in types or related places in utils so as to maintain cleanliness.